### ISSUE TITLE
cannot Enter or Backspace with v1.5.0 version

### ISSUE THREAD

**Issue by Agarwal-Nikhil** [neilsustc/vscode-markdown#214]
after press "enter or backspace" key, it pop "command 'markdown.extension.onEnterKey' not found" message, and cannot insert "newline"

install v1.1.0 version, last version don't have this issue. so it's regression.
use all default config

The problem goes away when I disable the extension.

**Comment by neilsustc** [comment]
Please try to
1. wait for several seconds after you open a Markdown file
2. if it doesn't work, open the develop console to see whether there are any error messages (menubar>help>toggle dev tools)

**Comment by CSU-NXY** [comment]
Hi, neilsustc,

I've met the same situation. Error message says 
`Activating extension 'yzhang.markdown-all-in-one' failed: Cannot find module '/usr/share/code/resources/app/extensions/markdown-language-features/out/slugify'.`

**Comment by neilsustc** [comment]
I guess you are running an old version of vscode (older than v1.24.0). You need to update it first.

**Comment by CSU-NXY** [comment]
You are right. I'm currently using vscode v1.22.2.  Updating vscode solves my problem. Thank you.
